### PR TITLE
Added examples to save and export menu items

### PR DESCRIPTION
**Pull request by Tech-FZ**
## What I added

Update: The export warning has now been removed and the Save and Export menu items now give examples for file types.

ApplicationFrameHost_Vy4G9PCG6w

The idea was requested via this issue by another user: https://github.com/PixiEditor/PixiEditor/issues/342

### CODE REVIEW

File: src/PixiEditor/Models/Dialogs/ExportFileDialog.cs [hunk 1]
```diff
@@ -78,11 +78,19 @@ public override bool ShowDialog()
...
+
         if (popup.DialogResult == true)
         {
             FileWidth = popup.SaveWidth;
             FileHeight = popup.SaveHeight;
             FilePath = popup.SavePath;
+
+            if (ChosenFormat.ToString() != "pixi")
+            {
+                ExportWarningPopup exportWarningPopup = new ExportWarningPopup();
```
Comment (flabbet) [inline src/PixiEditor/Models/Dialogs/ExportFileDialog.cs:90]: This is not a good solution. PixiEditor already has dialog popups that you can customize.

File: src/PixiEditor/Models/Dialogs/ExportFileDialog.cs [hunk 2]
```diff
@@ -78,11 +78,19 @@ public override bool ShowDialog()
...
         ExportFilePopup popup = new ExportFilePopup(FileWidth, FileHeight);
         popup.ShowDialog();
+
         if (popup.DialogResult == true)
         {
             FileWidth = popup.SaveWidth;
             FileHeight = popup.SaveHeight;
             FilePath = popup.SavePath;
+
+            if (ChosenFormat.ToString() != "pixi")
```
Comment (flabbet) [inline src/PixiEditor/Models/Dialogs/ExportFileDialog.cs:88]: Displaying popup each time user wants to export will be very annoying. This should be saved in editor data (user closed popup) so it only shows once.

**Review by flabbet — CHANGES_REQUESTED** [review]
I think this can be solved much easier. Most users know what .png file format is. So instead of popup explaining what export does, I'd simply replace "Export..." with "Export... (ex. .png, .jpg) and "Save" with "Save (.pixi). 

Another good solution would be to add a little question mark next to the text in context menu item with tooltip what the option does. Example: [Save (?)] where '?' is an icon

**Comment by Equbuxu** [comment]
> I think this can be solved much easier. Most users know what .png file format is. So instead of popup explaining what export does, I'd simply replace "Export..." with "Export... (ex. .png, .jpg) and "Save" with "Save (.pixi).
> 
> Another good solution would be to add a little question mark next to the text in context menu item with tooltip what the option does. Example: [Save (?)] where '?' is an icon

I have some comments on that, so let's discuss on discord first. As for the tests, we don't really have any tests for the UI right now, so don't worry about it.

File: src/PixiEditor/Views/MainWindow.xaml [hunk 3]
```diff
@@ -178,13 +178,13 @@
...
-                            Header="_Save"
+                            Header="_Save (.pixi)"
                             cmds:Menu.Command="PixiEditor.File.Save" />
                         <MenuItem
-                            Header="_Save As..."
+                            Header="_Save As... (.pixi)"
                             cmds:Menu.Command="PixiEditor.File.SaveAsNew" />
                         <MenuItem
-                            Header="_Export"
+                            Header="_Export (.png, .jpeg, .gif, .bmp)"
```
Comment (flabbet) [inline src/PixiEditor/Views/MainWindow.xaml:187]: This is a bit too long, I'd replace it with "_Export (.png etc.)"

**Review by flabbet — CHANGES_REQUESTED** [review]

File: src/PixiEditor/Views/MainWindow.xaml (hunk 3 above)
Comment (Equbuxu) [reply src/PixiEditor/Views/MainWindow.xaml:187]: ".png etc." sounds broken, maybe ".png, .jpg, etc."

Comment (flabbet) [reply src/PixiEditor/Views/MainWindow.xaml:187]: ok, ".png, .jpg, etc" then

**Review by flabbet — APPROVED** [review]

**Pull request merged**